data/json/furniture_and_terrain/furniture-storage.json

```diff
@@ -878,6 +878,100 @@
       "items": [ { "item": "scrap", "count": [ 8, 32 ] }, { "item": "water_faucet", "prob": 50 } ]
     }
   },
+  {
+    "type": "terrain",
```

Comment:
I'm pretty certain you wanted `"type": "furniture"`  here, and in following additions.